<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rozchody</class>
 <widget class="QMainWindow" name="Rozchody">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wydatnik - rozchody</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nazwa:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="edit_nazwa"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Kwota:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="edit_kwota"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Opis:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="edit_opis"/>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>Wydatek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Doch√≥d</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="button_dodaj">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="button_anuluj">
        <property name="text">
         <string>Anuluj</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>kwota</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="buttonCalendar">
        <property name="text">
         <string>Kalendarz</string>
        </property>
        <property name="icon">
         <iconset resource="icons/ikony.qrc">
          <normaloff>:/icon/calendar_icon_mini.png</normaloff>:/icon/calendar_icon_mini.png</iconset>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons/ikony.qrc"/>
 </resources>
 <connections/>
</ui>
